examples/sandbox/src/pages/elasticsearch/index.js

```diff
@@ -24,15 +24,21 @@ import {
 } from "@elastic/react-search-ui-views";
 import "@elastic/react-search-ui-views/lib/styles/styles.css";
 
-const connector = new ElasticSearchAPIConnector({
-  host:
-    process.env.REACT_ELASTICSEARCH_HOST ||
-    "https://search-ui-sandbox.es.us-central1.gcp.cloud.es.io:9243",
-  index: process.env.REACT_ELASTICSEARCH_INDEX || "national-parks",
-  apiKey:
-    process.env.REACT_ELASTICSEARCH_API_KEY ||
-    "SlUzdWE0QUJmN3VmYVF2Q0F6c0I6TklyWHFIZ3lTbHF6Yzc2eEtyeWFNdw=="
-});
+const connector = new ElasticSearchAPIConnector(
+  {
+    host:
+      process.env.REACT_ELASTICSEARCH_HOST ||
+      "https://search-ui-sandbox.es.us-central1.gcp.cloud.es.io:9243",
+    index: process.env.REACT_ELASTICSEARCH_INDEX || "national-parks",
+    apiKey:
+      process.env.REACT_ELASTICSEARCH_API_KEY ||
+      "SlUzdWE0QUJmN3VmYVF2Q0F6c0I6TklyWHFIZ3lTbHF6Yzc2eEtyeWFNdw=="
+  },
+  (esQuery) => {
+    console.log("postProcess Query Call", esQuery); // eslint-disable-line no-console
+    return esQuery;
+  }
```

Comment:
Since this is more of an advanced use case, what do you think about removing this from the sandbox? I worry that it might be considered required by users and will be copied to implementations.

Reply:
sure lets remove it